integration/helper/routeHelper.ts

```diff
@@ -0,0 +1,55 @@
+import request from 'supertest'
+
+export async function getOutOfBoundsReadAarch64(app: any, password: any, firstOffset: any, secondOffset: any) {
```

Comment:
it should be possible to give these types e.g. `app: Express, password: string` etc

Reply:
The app can. The rest can be it depends on how strictly typed we want. I guess we always want a password to be a string, the others could be provided as numbers or strings

Reply:
you can do `firstOffset: number | string`, just trying to remove all `any`